app/src/main/kotlin/com/wire/android/feature/auth/registration/CreateAccountFragment.kt

```diff
@@ -17,9 +18,9 @@ class CreateAccountFragment : Fragment(R.layout.fragment_create_account) {
 
     private fun initCreatePersonalAccount() {
         createPersonalAccountLayoutContainer.setOnClickListener {
-//            replaceFragment(
-//                R.id.activityCreateAccountLayoutContainer,
-//                CreatePersonalAccountFragment.newInstance())
+            replaceFragment(
```

Comment:
Same here with the style. If so, can you point me to where we got this from?

Reply:
As a rule of thumb, 140 characters per line is the standard.

Reply:
Also for brackets and expressions maybe it could be a good catch but for parameters, they should be in the same line.

I checked the Kotlin Lang official repository and I have not found anything like this. Here are 2 examples:
 - https://github.com/JetBrains/kotlin/blob/master/compiler/container/src/org/jetbrains/kotlin/container/Storage.kt
 - https://github.com/JetBrains/kotlin/blob/master/compiler/container/src/org/jetbrains/kotlin/container/Container.kt

Reply:
> As a rule of thumb, 140 characters per line is the standard.

[Here](https://developer.android.com/kotlin/style-guide#line_wrapping) it says 100 characters, we decided to use 120 before, I updated it to 140, but seems like there's no true "standard"

> Same here with the style. If so, can you point me to where we got this from?

I got it from copy paste :) I think the main concern here was line character count. Since we agreed on 140 now, we can write all in one line.
Really didn't put much attention, will try to write in a more Kotlin-esque way in the future.

> Also for brackets and expressions maybe it could be a good catch but for parameters, they should be in the same line.
> 
Again, line character count comes into play here. we should think this thoroughly. so if not all the parameters fits in 140 characters, should I cut if off in the middle? Sth like this?

```
method(param1, param2, param3, |
    param4)                    |
```

Reply:
Thanks for the points Gizem. The only cosmetic change was the `params` since this is how the official langauge is doing it, which for me looked weird.